workshops/templates/workshops/instructors.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content %}
-    {% crispy form form_helper %}
+    {% crispy form %}
 {% if persons == None %}
```

Comment:
Could you change that conditional? We don't need to check `if persons == None`, and it's not Pythonic.

Make that change in a separate commit, and explain in the message why you're making that change.